src/Sylius/Bundle/ApiBundle/Command/Payment/AddPaymentRequest.php

```diff
@@ -19,14 +19,14 @@
 class AddPaymentRequest implements IriToIdentifierConversionAwareInterface
 {
     public function __construct(
-        private string $paymentId,
+        private int|string $paymentId,
         private string $paymentMethodCode,
         private string $action,
         private mixed $payload = null,
```

Comment:
We could probably go with `public readonly` for all of these and remove the getters